driver/node/opera.go

```diff
@@ -248,3 +250,24 @@ func (n *OperaNode) RemovePeer(id driver.NodeID) error {
 func (n *OperaNode) Kill() error {
 	return n.container.SendSignal(docker.SigKill)
 }
+
+// GetNetworkDelayTo returns the median network delay to the given host.
+func (n *OperaNode) GetNetworkDelayTo(host string) (time.Duration, error) {
```

Comment:
I would call this method `GetLatency()`

Reply:
Renamed to `GetRoundTripTime()`.

Reply:
thanks, right, ping shows RTT, not Latency